README.md

```diff
@@ -81,6 +81,11 @@ Run end2end tests, for example `./out/Release/webnn_end2end_tests`.
  * The current implementation of XNNPACK and oneDNN backends is mainly for the investigation of WebNN [Operation Level Execution
 ](https://webmachinelearning.github.io/webnn/#usecase-op-level-exec) use case. So only a limited set of tests (such as of conv2d) is expected to pass.
 
+### Build and Run tests Status
```

Comment:
@BruceDai, although this PR has been merged, could you please send another PR to move the build status to the head as https://github.com/webmachinelearning/webnn-polyfill does. Thanks.

Reply:
Fixed by https://github.com/webmachinelearning/webnn-native/pull/32, thanks.